source/deployment/sso-saml.rst

```diff
@@ -24,6 +26,46 @@ Mattermost officially supports Okta, OneLogin and Microsoft ADFS as the identity
 
 In addition to the officially supported identity providers, you can also configure SAML for a custom IdP. For instance, customers have successfully set up Azure AD, DUO, PingFederate and SimpleSAMLphp as a custom IdPs. You can also set up MFA on top of your SAML provider for additional security.
 
+Using SAML Attributes to Apply Roles
+-------------------------------------
+
+You can use attributes to assign roles to specified users on login. To access the SAML attribute settings navigate to **System Console > SAML 2.0**. 
+
+##### Username Attribute
```

Comment:
Header formatting doesn't render correctly.

Reply:
Thanks for catching this - fixed!